_components/sidenav/sidenav.md

```diff
@@ -7,19 +7,11 @@ component:
 lead: "Hierarchical, vertical navigation to place at the side of a page."
 permalink: /components/side-navigation/
 redirect_from:
-- /sidenav/
-- /components/sidenav/
+  - /sidenav/
+  - /components/sidenav/
 subnav:
-- text: Preview
-  href: '#side-navigation-preview'
-- text: Code
-  href: '#side-navigation-code'
-- text: Guidance
-  href: '#side-navigation-guidance'
-- text: Package
-  href: '#side-navigation-package'
-- text: Latest updates
-  href: '#changelog'
+  - text: side navigation accessibility tests
```

Comment:
```suggestion
  - text: Side navigation accessibility tests
```

Reply:
Resolved in [9c30372](https://github.com/uswds/uswds-site/pull/2864/commits/9c3037216def30ecfc367e0adf4292a569722918)